### ISSUE TITLE
Error Reporting instrumentation error on Rails startup

### ISSUE THREAD

**Issue by hxiong388** [GoogleCloudPlatform/google-cloud-ruby#1011]
Right now when user tries to use Error Reporting instrumentation with Rails app, they get this error on Rails startup:
```
=> Booting Puma
=> Rails 5.0.0.1 application starting in production on http://0.0.0.0:3000
=> Run `rails server -h` for more startup options
Exiting
/usr/local/google/home/hexiong/rails_workspace/my_rails/gems/google-cloud-error_reporting/lib/google/cloud/error_reporting/rails.rb:62:in `block in <class:Railtie>': undefined local variable or method `error_reporting' for #<Google::Cloud::ErrorReporting::Railtie:0x0000000540c0e0> (NameError)
	from /usr/local/google/home/hexiong/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0.1/lib/rails/initializable.rb:30:in `instance_exec'
	from /usr/local/google/home/hexiong/.rvm/gems/ruby-2.3.1/gems/railties-5.0.0.1/lib/rails/initializable.rb:30:in `run'
...
```

I think this bug is introduced during the refactoring commits in PR #961.

**Comment by blowmage** [comment]
cc @hxiong388

**Comment by hxiong388** [comment]
Closed by [PR#1013](https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/1013)
